include/seqan3/io/sequence_file/format_fasta.hpp

```diff
@@ -135,7 +136,8 @@ class format_fasta
                                id_type && id,
                                qual_type && SEQAN3_DOXYGEN_ONLY(qualities))
     {
-        seqan3::ostreambuf_iterator stream_it{stream};
+        auto stream_view = seqan3::views::ostreambuf(stream);
+        auto stream_it = stream_view.begin();
```

Comment:
I don't think we need `views::ostreambuf` at all since there are no other "pure output views" currently and we never compare with end.

Just use the iterator directly like before!